docs/src/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,211 @@
+      )
+    }
+  }
+}
+```
+
+## Step 4: Instantiate PayKit iOS
+
+When you’re ready to authorize a payment using Cash App Pay,
+
+1. Instantiate the SDK with the Afterpay PayKit Client ID via `Afterpay.cashAppClientId`
+2. The SDK defaults to point to the production endpoint; for development, set the endpoint to `.sandbox`
+3. Add your observer to the PayKit SDK
+
+{: .note }
+> Ensure that the Afterpay SDK is configured per the [instructions][afterpay-configuration] before attempting to access `Afterpay.cashAppClientId`
+
+For example, from your checkout view controller that implements the `PayKitObserver` protocol, you might instantiate the SDK to be:
+
+``` swift
+private let sandboxClientID = Afterpay.cashAppClientId
+private lazy var sdk: PayKit = {
+    let sdk = PayKit(clientID: sandboxClientID, endpoint: .sandbox)
+    sdk.addObserver(self)
+    return sdk
+}()
+```
+## Step 5: Create a Customer Request
+
+You can create a customer request as soon as you know the amount you’d like to charge or if you'd like to create an on-file payment request. You’re encouraged to create this request as soon as your checkout view controller loads, so that your Customer can authorize the request without delay.
+
+### Step 5A: Sign the Order Token
+
+After retrieving the token from your server-to-server call, the order needs to be signed, so as to retrieve the JWT and associated data. This might look something like the below:
+
+``` swift
+let token = retrieveTokenFromServerToServerCall()
+Afterpay.signCashAppOrderToken(token) { result in
+  switch result {
+  case .success(let cashData):
+    // TODO: use `cashData` to create customer request (step 5B)
+  case .failed(let reason):
+    // TODO: handle failed signing
+  }
+}
+```
+
+### Step 5B: Create a PayKit Customer Request
+
+To charge a one-time payment, your create request call might look like this (in the below example, `cashData` is the response object that is returned in the `didCommenceCheckout` parameter from step 5A):
+
+``` swift
+paykit.createCustomerRequest(
+  params: CreateCustomerRequestParams(
+    actions: [
+      .oneTimePayment(
+        scopeID: cashData.brandId,
+        money: Money(
+          amount: cashData.amount,
+          currency: .USD
+        )
+      ),
+    ],
+    channel: .IN_APP,
+    redirectURL: URL(string: cashData.redirectUri)!,
+    referenceID: nil,
+    metadata: nil
+  )
+)
+```
+
+Your Observer’s state will change to `.creatingCustomerRequest`, then `.readyToAuthorize` with the created `CustomerRequest` struct as an associated value.
+
+## Step 6: Authorize the Customer Request
+
+### Step 6A: Add an Authorize Request Event to Cash App Pay button
+
+Once the SDK is in the `.readyToAuthorize` state, you can store the associated `CustomerRequest` and display a Cash App Pay button. When the Customer taps the button, you can authorize the customer request. Documentation for a Cash App Pay button component can be found [here][cash-button]{:target='_blank'}
```

Comment:
```suggestion
Once the SDK is in the `.readyToAuthorize` state, you can store the associated `CustomerRequest` and display a Cash App Pay button. When the Customer taps the button, you can authorize the customer request. See [CashAppPayButton][cash-button]{:target='_blank'} to to learn more about the Cash App Pay button component.
```

Reply:
@ScottAntonacAP you can add the same info as in the android docs "Currently, the Button provided by the SDK is unmanaged. This means that it’s a stylized button which isn’t aware of SDK events out-of-the-box. It is the developer’s responsibility to call the above method when the button is tapped and also manage any disabled and loading states."